graphql/schema/types/image.graphql

```diff
@@ -6,6 +6,13 @@ input PackageOpts {
   page: Int
 }
 
+input ToolchainOpts {
+    name: String
+    limit: Int
+    page: Int
+}
```

Comment:
nit. spacing; two spaces instead of four

Reply:
I think the Prettier plugin formats this for me on VSCode

<img width="591" alt="Screenshot 2024-07-29 at 11 49 16 AM" src="https://github.com/user-attachments/assets/44547ddf-4cfd-469e-a66f-dc0c90ed921a">